__tests__/app.test.js

```diff
@@ -167,3 +167,66 @@ describe("GET /api/reviews/:review_id/comments", () => {
             });
     });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+    test("201: POST adds a comment to the database for a specific review_id and responds with created comment", () => {
+        const newComment = {
+            username: "philippaclaire9",
+            body: "This game was more fun then the skirmish at Weathertop",
+        };
+
+        return request(app)
+            .post("/api/reviews/1/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body;
+                expect(comment).toEqual({
```

Comment:
Please use .toMatchObject() or .objectContaining() flexible matchers to give space to further expansion of this object.